src/components/AudioTrack/meta/AudioTrack.stories.mdx

```diff
@@ -110,3 +144,35 @@ The component is capable of playing audio in many formats, but this largely
 depends on the browser and engine. Refer to
 ['Can I use...'](https://caniuse.com/?search=audio%20format) for more up-to-date
 information on browser compatibility.
+
+## Single active player
+
+The audio players use a centralised store for holding the details of the active
```

Comment:
Works great, and thank you for Mozart!
It would be closer to the real usage if the `AudioTrack`s here used
`{ layout: 'row', size: 'm' }`, as in Related Audio.

Reply:
Resolved in b7baf27.